src/debug-macros.js

```diff
@@ -0,0 +1,150 @@
+/* eslint-env node */
+'use strict';
+const requireEsm = require('esm')(module);
```

Comment:
ESM typically slows stuff down quite abit, is it required?

Reply:
It's required if we want to have unstripped flags at runtime or ever want typescript support, but since I configured them to always strip though I'm thinking of eliminating the `-infra` directory entirely as debug-macros uses a virtualized module.

Reply:
virtualized them